modules/light-clients/07-tendermint/types/errors.go

```diff
@@ -10,16 +10,17 @@ const (
 
 // IBC tendermint client sentinel errors
 var (
-	ErrInvalidChainID         = sdkerrors.Register(SubModuleName, 2, "invalid chain-id")
-	ErrInvalidTrustingPeriod  = sdkerrors.Register(SubModuleName, 3, "invalid trusting period")
-	ErrInvalidUnbondingPeriod = sdkerrors.Register(SubModuleName, 4, "invalid unbonding period")
-	ErrInvalidHeaderHeight    = sdkerrors.Register(SubModuleName, 5, "invalid header height")
-	ErrInvalidHeader          = sdkerrors.Register(SubModuleName, 6, "invalid header")
-	ErrInvalidMaxClockDrift   = sdkerrors.Register(SubModuleName, 7, "invalid max clock drift")
-	ErrProcessedTimeNotFound  = sdkerrors.Register(SubModuleName, 8, "processed time not found")
-	ErrDelayPeriodNotPassed   = sdkerrors.Register(SubModuleName, 9, "packet-specified delay period has not been reached")
-	ErrTrustingPeriodExpired  = sdkerrors.Register(SubModuleName, 10, "time since latest trusted state has passed the trusting period")
-	ErrUnbondingPeriodExpired = sdkerrors.Register(SubModuleName, 11, "time since latest trusted state has passed the unbonding period")
-	ErrInvalidProofSpecs      = sdkerrors.Register(SubModuleName, 12, "invalid proof specs")
-	ErrInvalidValidatorSet    = sdkerrors.Register(SubModuleName, 13, "invalid validator set")
+	ErrInvalidChainID          = sdkerrors.Register(SubModuleName, 2, "invalid chain-id")
+	ErrInvalidTrustingPeriod   = sdkerrors.Register(SubModuleName, 3, "invalid trusting period")
+	ErrInvalidUnbondingPeriod  = sdkerrors.Register(SubModuleName, 4, "invalid unbonding period")
+	ErrInvalidHeaderHeight     = sdkerrors.Register(SubModuleName, 5, "invalid header height")
+	ErrInvalidHeader           = sdkerrors.Register(SubModuleName, 6, "invalid header")
+	ErrInvalidMaxClockDrift    = sdkerrors.Register(SubModuleName, 7, "invalid max clock drift")
+	ErrProcessedTimeNotFound   = sdkerrors.Register(SubModuleName, 8, "processed time not found")
+	ErrProcessedHeightNotFound = sdkerrors.Register(SubModuleName, 9, "processed height not found")
+	ErrDelayPeriodNotPassed    = sdkerrors.Register(SubModuleName, 10, "packet-specified delay period has not been reached")
```

Comment:
Going forward, I would consider changing error numbers a breaking change since others might be relying on the number value. I think understanding errors as a relayer is useful functionality. I think it is best to just append even if the ordering isn't logical

Reply:
Should I do this for this PR or for future ones?

Reply:
I think just future ones should be fine. I don't think relayers are using these values yet